tests/test_cache_memoize.py

```diff
@@ -350,3 +350,95 @@ def func_that_calls_runmeonce(*args):
         thread.join()
 
     assert len(calls_made) == 2
+
+
+def test_cache_exceptions():
+    calls_made = []
+
+    @cache_memoize(10, cache_exceptions=[Exception], prefix="cache_exceptions")
+    def raise_test_exception(exception):
+        calls_made.append(exception)
+        raise exception
+
+    try:
+        raise_test_exception(Exception)
+    except:
+        pass
+
+    try:
+        raise_test_exception(Exception)
+    except:
+        pass
+
+    assert len(calls_made) == 1
```

Comment:
is it worth being extra paranoid and asserting the number of calls after every function call?

Reply:
I dont' follow you here. Could you elaborate?

Reply:
So when you check `assert len(calls_made) == 1` this at the very end, you don't test exactly when the call was made. Was it made the first time you called the function (hopefully)? Or the second?

So I was suggesting something like:

```
with pytest.raises(Exception):
    raise_test_exception(Exception)

assert len(calls_made) == 1

with pytest.raises(Exception):
    raise_test_exception(Exception)

assert len(calls_made) == 1
```

but honestly, this is a bit of a nitpick :)